### ISSUE TITLE
Crash when image contains cyrillic characters in name

### ISSUE THREAD

**Issue by osipxd** [RedMadRobot/figma-export#44]
```
$ figma-export images "АБВГД"
2020-11-27T09:19:02+0300 info: Using FigmaExport to export images to Android Studio project.
2020-11-27T09:19:02+0300 info: Fetching images info from Figma. Please wait...
2020-11-27T09:19:06+0300 info: Processing images...
2020-11-27T09:19:06+0300 info: Downloading remote files...
fish: 'figma-export images "…' terminated by signal SIGILL (Illegal instruction)
```

I think figma-export should throw an exception instead of a silent crash

**Comment by subdan** [comment]
Thanks. I will fix it by the end of the week.

**Comment by subdan** [comment]
I've released new version 0.18.1 which fixes this issue

**Comment by osipxd** [comment]
Blazing fast!
